src/lib/systemlib/system_params.c

```diff
@@ -144,9 +143,10 @@ PARAM_DEFINE_INT32(SYS_COMPANION, 0);
 /**
  * Parameter version
  *
- * This monotonically increasing number encodes the parameter compatibility set.
- * whenever it increases parameters might not be backwards compatible and
- * ground control stations should suggest a fresh configuration.
+ * This is used internally only: an airframe configuration might set an expected
```

Comment:
What do you mean with "internally only"?

Reply:
In terms of that a user does not need to change this.